tests/sentry/receivers/test_superuser.py

```diff
@@ -0,0 +1,53 @@
+from sentry.auth.superuser import is_active_superuser
+from sentry.models import User
+from sentry.receivers.superuser import disable_superuser, enable_superuser
+from sentry.testutils import TestCase
+
+
+class SuperuserReceiverTest(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.superuser = User(is_superuser=True, email="test@sentry.io")
+        self.non_superuser = User(is_superuser=False, email="test@sentry.io")
+
+        self.superuser_request = self.make_request(user=self.superuser)
+        self.non_superuser_request = self.make_request(user=self.non_superuser)
+
+    def test_enable_superuser_when_self_hosted_superuser(self):
```

Comment:
nit: can we indicate `superuser` vs `non_superuser` with `__` as opposed to `_`?
Makes the test titles a bit more clear

Reply:
Or alternatively - you can smash the negative tests together and just have multiple assertions for the different cases 🤷